docs/configuration.md

```diff
@@ -92,28 +92,28 @@ Sugoi-api implements springdoc with full customization allowed
+| fr.insee.sugoi.security.ldap-account-managment-enabled     |                                                                                                                        |               |         |
+| fr.insee.sugoi.security.ldap-account-managment-url         |                                                                                                                        |               |         |
+| fr.insee.sugoi.security.ldap-account-managment-user-base   |                                                                                                                        |               |         |
+| fr.insee.sugoi.security.ldap-account-managment-groupe-base |                                                                                                                        |               |         |
+| fr.insee.sugoi.security.bearer-authentication-enabled      |                                                                                                                        |               |         |
+| spring.security.oauth2.resourceserver.jwt.jwk-set-uri      |                                                                                                                        |               |         |
+| fr.insee.sugoi.api.old.regexp.role.consultant              |                                                                                                                        |               |         |
+| fr.insee.sugoi.api.old.regexp.role.gestionnaire            |                                                                                                                        |               |         |
+| fr.insee.sugoi.api.old.regexp.role.admin                   |                                                                                                                        |               |         |
+| fr.insee.sugoi.api.old.enable.preauthorize                 |                                                                                                                        |               |         |
+| fr.insee.sugoi.api.regexp.role.reader                      |                                                                                                                        |               |         |
+| fr.insee.sugoi.api.regexp.role.writer                      |                                                                                                                        |               |         |
+| fr.insee.sugoi.api.regexp.role.admin                       |                                                                                                                        |               |         |
+| fr.insee.sugoi.api.regexp.role.app.manager                 | Default pattern to use when searching manager group for application. Application name should be passed via {appliname} |               |         |
```

Comment:
I mistook this variable with the variable `fr.insee.sugoi.ldap.default.group_manager_source_pattern` . Can you move the description to this variable and change the default ref in realm-configuration.md ?

Reply:
Done ✔️ @CChemin  👍